playground/cli/jsonToPDF/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "fetch-json-data",
+  "version": "1.0.0",
+  "description": "convert json data to pdf",
+  "main": "\u001b[A\u001b[B",
```

Comment:
not sure what this means :)

Reply:
Typo mistake , let me correct and push again